test/integration/test_open_analyse_save_load_project.c

```diff
@@ -37,7 +37,7 @@ bool test_open_analyse_save() {
 	size_t functions_count_expect = rz_list_length(functions);
 
 	// 4. Save into the project
-	if (!rz_file_exists(".tmp/")) {
+	if (!rz_file_is_directory(".tmp/")) {
```

Comment:
You can remove this commit, since ret2libc has fixed this on another PR which is merged

Reply:
Rebased to `dev`.